app/lib/kafka.server.ts

```diff
@@ -44,13 +44,23 @@ export let send: (topic: string, value: string) => Promise<void>
 // testing.
 if (process.env.ARC_SANDBOX) {
   send = async (topic, value) => {
-    setOidcHttpOptions()
-    const producer = kafka.producer()
-    await producer.connect()
     try {
-      await producer.send({ topic, messages: [{ value }] })
-    } finally {
-      await producer.disconnect()
+      setOidcHttpOptions()
+      const producer = kafka.producer()
+      await producer.connect()
+      try {
+        await producer.send({ topic, messages: [{ value }] })
+      } finally {
+        await producer.disconnect()
+      }
+    } catch (e) {
+      if (e instanceof KafkaJSNonRetriableError) {
+        console.log(
+          'Kafka is not correctly configured. This would be an error on Production, but continuing on dev'
+        )
+      } else {
```

Comment:
`if (e instance KafkaJSNonRetriableError && /* we are in sandbox mode */)`

You can find examples of how to evaluate `/* we are in sandbox mode */` in the code base.